src/test/scala/com/stratio/hermes/helpers/RandomHelperTest.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.hermes.helpers
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FlatSpec, Matchers}
+
+@RunWith(classOf[JUnitRunner])
+class RandomHelperTest  extends FlatSpec with Matchers {
+
+  "A RandomHelper" should "generates an element from a generic list" in {
+    RandomHelper.randomElementFromAList(List(1, 2, 3)) should not be (None)
```

Comment:
Instead of checking for "not be None" in my opinion it might be better to check whether the method generates an element from the list, something like: RandomHelper.randomElementFromAList(List(1, 2, 3)) should contain oneOf (1, 2, 3)